resources/lib/utils.py

```diff
@@ -78,7 +78,10 @@ def decode_data(data):
     ''' Decode data coming from a notification event '''
     data = json.loads(data)
     if data:
-        return to_unicode(json.loads(unhexlify(data[0])))
+        json_data = unhexlify(data[0])
+        if isinstance(json_data, bytes):
```

Comment:
Can you add a small comment that this is required for Python 3.5 and older ?
At least we know why this is needed and can remove it when it no longer is needed.

Reply:
Comment is added 👍